libs/core/src/lib/token/tokenizer.component.ts

```diff
@@ -419,6 +428,28 @@ export class TokenizerComponent implements AfterViewInit, OnDestroy, CssClassBui
         return totalTokenWidth;
     }
 
+    /**
+     * @hidden
+     * Check whether user intends to go back to the previous token
+     **/
+    private _goBackRequested($event: KeyboardEvent, rtl: boolean): boolean {
+        if (!rtl) {
```

Comment:
Can be simplified like:
```typescript
const isBackButton = isRtl ? [RIGHT_ARROW, DOWN_ARROW] : [LEFT_ARROW, UP_ARROW];

return KeyUtil.isKeyCode($event, isBackButton);
```

Reply:
Same logic can be applied for `_goForwardRequested`

Reply:
And maybe rename methods to `_isGoBackRequested` and `_isGoForwardRequested` since these methods check on whether the action being requested and not fire any action by itself.